src/global/Constants.h

```diff
@@ -245,3 +246,6 @@ constexpr inline size_t NUM_SORT_THREADS = 4;
 constexpr inline std::string_view EMPH_ON = "\033[1m";
 /// ANSI escape sequence to print "normal" text again in the console.
 constexpr inline std::string_view EMPH_OFF = "\033[22m";
+
+// Counter for internal Blank Node Ids.
+inline std::atomic_uint64_t NEXT_NEWBLANKNODEINDEX{0};
```

Comment:
Better call them (here and everywhere `local` blank nodes (not `new`). Then it is consistent with the `local` vocab, which is a similar concept.